webapp/store/snap_details_views.py

```diff
@@ -17,6 +17,52 @@
 )
 from webapp.markdown import parse_markdown_description
 
+import os
```

Comment:
in `webapp/helpers.py` there's a _get_yaml function. This is where I added the `get_livestreams` stuff. Might be worth moving some of this stuff to there (specifically the yaml opening stuff)

Reply:
Also https://github.com/canonical-web-and-design/snapcraft.io/pull/1948 ....

Reply:
@jpmartinspt Has separate task for read file and yaml refactoring. Will update feature branch once that is merged.